meetings/2022-11-16.md

```diff
@@ -45,45 +57,103 @@
+https://mathstodon.xyz/@bblfish/109326189343436621
+* HS: See especially the [Mastodon docs](https://docs.joinmastodon.org/spec/security/#http).
+* HS: The [HTTP signature](https://www.ietf.org/archive/id/draft-ietf-httpbis-message-signatures-13.html) is progressing to the finish line, so I'm going to start doing a rewrite and cleaning up. If people are interested we can meet in the Web authentication panel. 
+* HS: I will be working on improving the Solid Http Sig spec, which makes use of the ietf docs.
+https://github.com/bblfish/authentication-panel/blob/main/proposals/HttpSignature.md
 
 
 ### Storage Description
 URL: https://solidproject.org/ED/protocol#server-storage-description
 
-* SC: On discovery of storage description resource via `http://www.w3.org/ns/solid#storageDescription`
+* SC: On discovery of storage description resource via `http://www.w3.org/ns/solid/terms#storageDescription`
 * SC: Background: https://github.com/solid/specification/issues/355
 * SC: Currently also used to discover information about Notification Channels: https://solid.github.io/notifications/protocol#server-link-storage-notification-channel
 * SC: but.. we need solid implementation feedback. Commitment to implement.
 
+* SC: A high overview: In the editor's draft we have a link to a requirement (link?) issue #355. If you haven't seen it please do. 
+* SC: The term is proposed to be in Solid Terms vocabulary. Proposing to include the property to describe the storage description in the Notifications Protocol. What we're saying is when a client wants to find out information about notification channel, 2 ways: one is through storage description. We agreed we will keep that in the notifications protocol spec, now it remains there and published eventually. If that changes we will need to consider another way. Raising as topic because it's part of the Protocol where there's uses, on paper it sounds great, but that is not always enough to keep it in the spec, specially an ED, a good signal would be to have multiple implementations, or come up with an alternative proposal that meets the same use cases. Hoping that if you haven't come across it you'd give it a look. Relatively easy to implement, but we still needs to see implementations (servers or applications) making use of that discovery, or a commitment to implement. Signals of that sort are important. Even if you say this makes no sense, that's as useful as commitment to implement. That's how we know if it needs to be on the spec. When we have tests for these things and need to show implementation reports, the results will show whether that particular implementation agreed to use that, and if not implemented, we can remove from the spec. That's why we go through this process. That goes for any feature or solutions that we have in the spec. If no one will implement a certain feature, we need to focus on low hanging fruit.
```

Comment:
```suggestion
* SC: The term is proposed to be in Solid Terms vocabulary. Proposing to include the property to describe the storage description in the Notifications Protocol. We're saying that when a client wants to find out information about notification channel, there are two ways: one is through storage description. We agreed we will keep that in the notifications protocol spec; now it remains there and published eventually. If that changes, we will need to consider another way. Raising as topic because it's part of the Protocol where there's uses; on paper it sounds great, but that is not always enough to keep it in the spec, specially an ED. A good signal would be to have multiple implementations, or come up with an alternative proposal that meets the same use cases. Hoping that if you haven't come across it you'd give it a look. Relatively easy to implement, but we still need to see implementations (servers or applications) making use of that discovery, or a commitment to implement. Signals of that sort are important. Even if you say this makes no sense, that's as useful as commitment to implement. That's how we know if it needs to be on the spec. When we have tests for these things and need to show implementation reports, the results will show whether that particular implementation agreed to use that, and if not implemented, we can remove from the spec. That's why we go through this process. That goes for any feature or solutions that we have in the spec. If no one will implement a certain feature, we need to focus on low hanging fruit.
```